dcicutils/license_utils.py

```diff
@@ -646,6 +954,109 @@ class C4InfrastructureLicenseChecker(LicenseChecker):
         'Zope Public License',
     ]
 
+    EXCEPTIONS = {
```

Comment:
Should this be read from a file?

Reply:
I'm going to look into that. The trade-off is that these are really things that should be matters of fixed policy, at least insofar as the validator functions are, and people should not be thinking they can just casually edit this stuff to make it go away. It requires a lot of thought about legalities. But, that said, the command line form allows someone to explore whether they're compliant and to work toward getting things in order. For that it can be useful to evolve something and propose it as a coherent whole. So it's not a technical matter but a social, workflow, and legal issue. I'll do a little more functionality on that, though.

Reply:
This got done and is now merged to this PR.